roles/manifest/README.md

```diff
@@ -0,0 +1,69 @@
+theforeman.foreman.manifest
+===============================================
+
+Upload Subscription Manifest to Katello
+
+Requirements
+------------
+
+- `theforeman.forman` collection
+
+Role Variables
+--------------
+
+- `server_url`: URL of the Foreman server.
+- `username`: Username accessing the Foreman server.
+- `password`: Password of the user accessing the Foreman server.
+- `validate_certs`: Whether or not to verify the TLS certificates of the Foreman server.
+- `organization`: Organization to upload Manifest to.
```

Comment:
Given this PR introduces them, could we add and document the variables above here also here -- https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md

Reply:
:+1: